src/Mvc/Mvc.Testing/src/Handlers/RedirectHandler.cs

```diff
@@ -173,6 +173,7 @@ private static async Task<(Stream originalBody, Stream copy)> CopyBody(HttpReque
         private bool IsRedirect(HttpResponseMessage response) =>
             response.StatusCode == HttpStatusCode.MovedPermanently ||
                 response.StatusCode == HttpStatusCode.Redirect ||
+                response.StatusCode == HttpStatusCode.RedirectMethod ||
```

Comment:
Do we also need to update `ShouldKeepVerb` in line 152?

Reply:
I don't think so.
The client following the 303 redirection must use GET method to get redirected page no matter the original method was:

- https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303
- https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

I must confess I didn't notice the `ShouldKeepVerb` and didn't take it into account before my PR...